index.html

```diff
@@ -419,6 +419,454 @@ <h2>Capabilities Are Safer</h2>
+          long chain attacks. A verifier SHOULD limit the length of the
+          capability chain to 10. [exposition on why 10 / link to security
+          section].
+        </p>
+
+        <p>
+          A root zcap MUST have an invocationTarget that is a string that
+          expresses a URI. The invocation target identifies where the zcap may
+          be invoked. A verifier MUST ensure that the invocationTarget either
+          matches the invocationTarget in the parent capability or, if
+          invocation target attenuation [link] is permitted, that it has the
+          invocationTarget from the parent capability as a prefix. A prefix is
+          defined as a base URI and parent path (and optional query) (i.e.,
+          / delimited and ?/& delimited) [more rigorous definition].
+        </p>
+
+        <p>
+          A delegated zcap MUST have a controller that is a string or an array
+          of strings that each express a URI that identifies a controller for
+          the delegated zcap. The controller (or controllers) may take any
+          allowed actions [link] with the invocation target (that are supported
+          by the verifier) by invoking the delegated zcap. The controller (or
+          controllers) may create delegated zcaps from the delegated zcap.
+          [Note: As with other data model sections in W3C specs, every property
+          of a zcap should be called out in its own subsection along with the
+          rules for the property.]
+        </p>
+
+        <p>
+          A delegated zcap MUST have an expires field that expresses an XSD
+          date-time (Note: The JavaScript new Date().toISOString() code can
+          produce such a date representation, though it is preferred to remove
+          millisecond precision via new Date().toISOString().slice(0, -5) +
+          'Z').
+        </p>
+
+        <p>
+          A verifier MUST ensure that an invoked delegated zcap has not expired.
+          A verifier MUST ensure that a delegated zcap's expiration date-time is
+          not less restrictive than its parent capability's expiration
+          date-time, if present (a root zcap does not have an expiration
+          date-time).
+        </p>
+
+        <p>
+          A verifier SHOULD ensure that an invoked delegated zcap does not have
+          an expiration date-time that is more than, e.g., three months in the
+          future [TODO: exposition on why 3 months?]. This is because a verifier
+          MUST store revoked zcaps [link to revocation] until they expire to
+          prevent their use. A delegated zcap with an expiration date that is
+          unreasonably far into the future will have to be stored for an
+          unreasonable period of time to prevent its invocation. There are
+          other mitigation strategies here such as considering all zcaps
+          delegated from a particular controller as revoked or full key
+          revocation.
```

Comment:
```suggestion
          A verifier SHOULD ensure that an invoked delegated zcap does not have
          an expiration date-time that is more than three months in the
          future. [TODO: exposition on why 3 months?] This is because a verifier
          MUST store revoked zcaps [link to revocation] until they expire, to
          prevent their use. A delegated zcap with an expiration date that is
          unreasonably far into the future will have to be stored for an
          unreasonable period of time to prevent its invocation. There are
          other mitigation strategies here, such as considering all zcaps
          delegated from a particular controller as revoked, or full key
          revocation.
```